x/mongo/driver/topology/server.go

```diff
@@ -434,9 +467,9 @@ func (s *Server) ProcessError(err error, conn driver.Connection) driver.ProcessE
 
 		res := driver.ServerMarkedUnknown
 		// If the node is shutting down or is older than 4.2, we synchronously clear the pool
-		if wcerr.NodeIsShuttingDown() || desc.WireVersion == nil || desc.WireVersion.Max < 8 {
+		if wcerr.NodeIsShuttingDown() || wireVersion == nil || wireVersion.Max < 8 {
```

Comment:
What are your thoughts on creating a constant for "8"? Even though comparing wireVersions to their literal number is commonplace in the driver, it is a magic number which makes it sort of hard to interpret.

```go
const wireVersionGEQ42 = 8
```